recipe/meta.yaml

```diff
@@ -1,52 +1,57 @@
 {% set name = "httpstan" %}
-{% set version = "4.6.1" %}
+{% set version = "4.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/stan-dev/httpstan/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 703e5e04e60651e0004574bb9695827d759fd13eb0d6bd67f827c1bfa0a1fd31
+  sha256: 6b15a07557715e79e6fd66993930003b270f8b8b0c9e65f84978afe5e6bb3047
 
 build:
-  number: 3
-  script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [py<37] 
-  
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  skip: True  # [py<310]
+
 
 requirements:
   build:
```

Comment:
I think you can remove requirements/build and pybind11 from host. 
Unless I am mistaken there is no compiled code in this package.